SConstruct

```diff
@@ -238,6 +238,28 @@ preSfm = command("sfm", [skymap, transmissionCurvesTarget],
 env.Depends(preSfm, refcat)
 sfm = {(data.visit, data.ccd): data.sfm(env) for data in sum(allData.values(), [])}
 
+# Sky correction
+def processSkyCorr(dataList):
+    """Generate sky corrections"""
+    visitDataLists = defaultdict(list)
+    for ff in allData:
+        for data in allData[ff]:
+            visitDataLists[data.visit].append(data)
+    preSkyCorr = command("skyCorr", skymap,
+                         getExecutable("pipe_drivers", "skyCorrection.py") + " " + PROC + " " + STDARGS +
+                         " --batch-type=smp --cores=1")
```

Comment:
Why `smp` rather than `none`?

Reply:
With `--cores=1` there's not a lot of difference, but `none` might produce more useful error messages so I'll go with that.